blog.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Blog</title>
+  </head>
+
+  <body>
+    <header class="main-header">
+      <img class="adoptadog-logo" src="images/adoptadog-logo.svg" alt="Adoptadog logo">
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <img class="cart-icon" src="images/cart-icon.svg" alt="Shopping cart icon">
+      <p>$1234.56</p>
+      <a href="checkout.html">Checkout</a>
+    </header>
+
+    <nav class="main-nav">
+      <a href="index.html" target="_blank">Home  /</a>
+      <a href="dogs.html" target="_blank">Dogs  /</a>
+      <a href="blog.html" target="_blank">Blog</a>
+    </nav>
+
+    <section>
+        <h2>Adoptadog Blog</h2>
+        <article class="traveling">
```

Comment:
You'll probably want to use the same class for each of these articles for ease in styling.